pip_outdated/print_outdated.py

```diff
@@ -32,14 +32,14 @@ def colored_latest():
         colored_wanted(),
         colored_latest()
     ]
-
-def print_outdated(outdates, quiet: bool):
+    
+async def print_outdated(outdates, quiet: bool):
     colorama.init()
     
     data = [["Name", "Installed", "Wanted", "Latest"]]
     count = 0
-    for count, outdate in enumerate(outdates, 1):
-        row = make_row(outdate)
+    for count, outdate in enumerate(list(outdates), 1):
```

Comment:
casting to a list here doesn't do anything, so it can just be

```python
for count, outdate in enumerate(outdates, 1):
```

Reply:
I was wrong, so ignore this. The list is necessary to make sure you exhaust the generator, in order to make sure all of the tasks are started. If not, you would actually only start one at a time as you are awaiting the result of each one sequentially.